src/util/misc/mergeClipPaths.ts

```diff
@@ -20,21 +21,15 @@ import { multiplyTransformMatrices, invertTransform } from './matrix';
  * @param {fabric.Object} c2
  * @returns {fabric.Object} merged clip path
  */
-export const mergeClipPaths = (c1: any, c2: any) => {
+export const mergeClipPaths = (c1: TObject, c2: TObject) => {
   let a = c1, b = c2;
   if (a.inverted && !b.inverted) {
     //  case (2)
     a = c2;
     b = c1;
   }
   //  `b` becomes `a`'s clip path so we transform `b` to `a` coordinate plane
-  applyTransformToObject(
-    b,
-    multiplyTransformMatrices(
-      invertTransform(a.calcTransformMatrix()),
-      b.calcTransformMatrix()
-    )
-  );
+  sendObjectToPlane(b, b.group?.calcTransformMatrix(), a.calcTransformMatrix());
```

Comment:
```suggestion
  sendObjectToPlane(b, b.group?.calcTransformMatrix() ?? IMatrix, a.calcTransformMatrix());
```
and let's remove the null usecase.

Reply:
this resolves to undefined so it is covered

Reply:
oh i thought it resolved to null, i guess too much nulls in TS in the project where i work on